runtime/common/src/lib.rs

```diff
@@ -226,24 +239,49 @@ pub mod changes {
 	}
 
 	/// Used for building CfgChanges in pallet-loans
-	impl<T: pallet_loans::Config> From<LoansChange<T>> for RuntimeChange<T> {
+	impl<T: pallet_loans::Config + frame_system::Config> From<LoansChange<T>> for RuntimeChange<T> {
 		fn from(loan_change: LoansChange<T>) -> RuntimeChange<T> {
 			RuntimeChange::Loan(loan_change)
 		}
 	}
 
+	/// Used for building CfgChanges in pallet-pool-fees
+	impl<T: pallet_loans::Config + frame_system::Config> From<PoolFeesChange<T::AccountId, Ratio>>
+		for RuntimeChange<T>
+	{
+		fn from(pool_fees_change: PoolFeesChange<T::AccountId, Ratio>) -> RuntimeChange<T> {
+			RuntimeChange::PoolFee(pool_fees_change)
+		}
+	}
+
 	/// Used for recovering LoanChange in pallet-loans
-	impl<T: pallet_loans::Config> TryInto<LoansChange<T>> for RuntimeChange<T> {
+	impl<T: pallet_loans::Config + frame_system::Config> TryInto<LoansChange<T>> for RuntimeChange<T> {
 		type Error = DispatchError;
 
 		fn try_into(self) -> Result<LoansChange<T>, DispatchError> {
-			let RuntimeChange::Loan(loan_change) = self;
-			Ok(loan_change)
+			match self {
+				RuntimeChange::Loan(change) => Ok(change),
+				_ => Err(DispatchError::Other("Not a loans change")),
+			}
+		}
+	}
+
+	/// Used for recovering PoolFeeChange in pallet-pool-fees
+	impl<T: pallet_loans::Config + frame_system::Config>
+		TryInto<PoolFeesChange<T::AccountId, Ratio>> for RuntimeChange<T>
+	{
+		type Error = DispatchError;
+
+		fn try_into(self) -> Result<PoolFeesChange<T::AccountId, Ratio>, DispatchError> {
+			match self {
+				RuntimeChange::PoolFee(pool_fees_change) => Ok(pool_fees_change),
+				_ => Err(DispatchError::Other("Not a pool fees change")),
+			}
 		}
 	}
 
 	pub mod fast {
-		use pallet_pool_system::pool_types::changes::Requirement;
+		use pallet_pool_system::pool_types::{changes::Requirement, PoolChangeProposal};
```

Comment:
Sorry for the boilerplate here. Probably you need to add the same in the `fast` mode to support development runtime.

Reply:
I'm thinking that maybe this can be fixed by moving the "fast" action into the pool system instead of doing it as a wrapper over `RuntimeChange`; it probably makes more sense. I'll try to make a new PR with this ASAP

Reply:
Thanks for raising this. I might have eventually forgotten about it 😅 I first wanted to finish the feature MVP and then deal with the leftover bits.

> I'm thinking that maybe this can be fixed by moving the "fast" action into the pool system instead of doing it as a wrapper over RuntimeChange; it probably makes more sense. I'll try to make a new PR with this ASAP

That would be awesome! The ETA for merging this PR here is EOY btw.

Reply:
Well, it was a fast move 😄 . It's here: https://github.com/centrifuge/centrifuge-chain/pull/1634

So, I think after you rebase, you simply need to skip that part.